### DISCUSSION TITLE
Page not responding

### DISCUSSION THREAD

**Discussion by gin-lsl** [community/community#37445] [Other Feature Feedback, Questions, & Ideas]
When I visit [GitHub/vuepress](https://github.com/vuejs/vuepress/blob/master/packages/%40vuepress/plugin-medium-zoom/index.js) , the page is unresponsive and the Chrome task manager shows CPU 100%.

**Comment by gin-lsl** [comment]
I tried Chrome, Edge, Firefox, none of them respond

**Comment by ossanna16** [comment]
Hi there @gin-lsl and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
